execution/src/posix/execution.cc

```diff
@@ -314,13 +314,30 @@ namespace leatherman { namespace execution {
         }
 
         // Close all open file descriptors above stderr
-#ifdef HAS_CLOSEFROM
+#if defined(HAS_CLOSEFROM)
         closefrom(STDERR_FILENO + 1);
 #else
+  #if defined(HAS_PROC_PID)
```

Comment:
I think this is no longer needed as there is already check of directory existence in code